browsers/safari_ios.json

```
@@ -6,142 +6,170 @@
         "1": {
           "status": "retired",
           "engine": "WebKit",
-          "engine_version": "522.11"
+          "engine_version": "522.11",
+          "release_date": "2007-06-29"
         },
         "2": {
           "status": "retired",
           "engine": "WebKit",
-          "engine_version": "525.18"
+          "engine_version": "525.18",
+          "release_date": "2008-07-11"
         },
         "3": {
           "status": "retired",
           "engine": "WebKit",
-          "engine_version": "528.18"
+          "engine_version": "528.18",
+          "release_date": "2009-06-17"
         },
         "3.1": {
           "status": "retired",
           "engine": "WebKit",
-          "engine_version": "528.18"
+          "engine_version": "528.18",
+          "release_date": "2009-09-09"
         },
         "3.2": {
           "status": "retired",
           "engine": "WebKit",
-          "engine_version": "531.21"
+          "engine_version": "531.21",
+          "release_date": "2010-04-03"
         },
         "4": {
           "status": "retired",
           "engine": "WebKit",
-          "engine_version": "532.9"
+          "engine_version": "532.9",
+          "release_date": "2010-06-21"
         },
         "4.2": {
           "status": "retired",
           "engine": "WebKit",
-          "engine_version": "533.17"
+          "engine_version": "533.17",
+          "release_date": "2010-10-17"
```

I can't figure out where this release date came from. It looks like 4.2.1 was the first actual release and the linked sources give 2010-11-22 as the release date

```suggestion
          "release_date": "2010-11-22"
```